sdk/src/main/java/com/appsci/panda/sdk/ui/SubscriptionFragment.kt

```diff
@@ -259,13 +258,24 @@ class SubscriptionFragment : Fragment() {
                         Timber.d("setPayload result $it")
                     }
                 }
+                /**
+                 * @Deprecated
+                 * used for backward compatibility
+                 */
                 lifecycleScope.launchWhenStarted {
-                    if (!showTrial.await()) {
+                    if (!showTrialCompletable.await()) {
                         _binding?.webView?.evaluateJavascript("removeTrialUi();") {
                             Timber.d("removeTrialUi result $it")
                         }
                     }
                 }
+
+                lifecycleScope.launchWhenStarted {
+                    val showTrial = showTrialCompletable.await()
+                    _binding?.webView?.evaluateJavascript("showTrial($showTrial);") {
```

Comment:
please add this to doc

Reply:
https://www.notion.so/boosters/Panda-Android-SDK-267d15884122422cb5297a3a4717eef5?pvs=4